source/active_support_core_extensions.md

```diff
@@ -850,7 +850,9 @@ NOTE: Определено в `active_support/core_ext/module/delegation.rb`.
 
 Метод `redefine_method` предотвращает такое потенциальное предупреждение, предварительно убирая существующий метод, если нужно.
 
-NOTE: Определено в `active_support/core_ext/module/remove_method.rb`.
+Также можно использовать `silence_redefinition_of_method`, если нужно определить метод замены себя (потому что используется `delegate`, например).
```

Comment:
Оригинал:
You can also use silence_redefinition_of_method **if you need to define the replacement method yourself** (because you're using delegate, for example).

Reply:
@Mik-die 
А может здесь перевести так:
"если необходимо определить исключительно заменяющий метод"
?

Reply:
не, так смысл тоже теряется